src/Storages/ExternalStream/ExternalStreamSettings.h

```diff
@@ -15,6 +15,7 @@ class ASTStorage;
     M(String, security_protocol, "plaintext", "The protocol to connection external logstore", 0) \
     M(String, username, "", "The username of external logstore", 0) \
     M(String, password, "", "The password of external logstore", 0) \
+    M(String, ssl_ca_cert_file, "", "The path of ssl ca cert file", 0) \
```

Comment:
Maybe this can be enhanced in another PR, but for external streams, we can't assume users have access to proton, thus specifying the CA file location is not always feasible.